src/pages/grad-app-assistance.mdx

```diff
@@ -0,0 +1,61 @@
+import { Button } from "../components/button";
+import { VSpace } from "../components/spacing";
+import { faPaperPlane } from "@fortawesome/free-solid-svg-icons/faPaperPlane";
+
+# Grad School Application Funding
+<br />
+<br />
+
+## What is this?
+
+oSTEM and Queer in AI are working to end application and test fees as a barrier to queer 
+STEM scholars applying to graduate programs. Queer students and scholars are underrepresented 
+and marginalized within STEM fields. Applying to graduate schools typically costs many hundreds 
+of dollars which poses a major barrier to queer students, who are disproportionately affected by 
+economic hardship. Queer students are often forced to apply to a very limited number of schools, 
+take out costly loans, or not apply at all.
+
+## Apply for financial assistance
+
+The Graduate Applications Financial Aid Program will provide up to **$750 per individual** to cover 
+application or test fees for queer people applying to STEM graduate programs. Applicants from all 
+countries are welcomed.  This year's program will close on June 1st, 2021.
+
+To apply, click the link below and create a _Financial Aid_ submission.
+
+<center>
+
+<VSpace size={"2rem"} />
+<Button icon={faPaperPlane} enabled onClick={"https://cmt3.research.microsoft.com/User/Login?ReturnUrl=%2FQAI2020%2FSubmission%2FIndex"}>
+  Apply Now 
+</Button>
+
+</center>
+<br />
+
+## How can I support this program?
+
+We are accepting donations to support this program!
+
+<center>
```

Comment:
The `<center>` tag is an old historical thing, and I'd recommend using a `<div />` with padding instead of a `<VSpace />` here.

You should probably use

```
<div
  className="p-4 text-center"
>
(all the other stuff goes here)
</div>
```

The `p-4` means to have a padding of `4` (the units are arbitrary, but `4` corresponds to `1rem` which usually corresponds to `16px`), and `text-center` will center the content within the div.

Reply:
Also, same for above.